app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    render plain: produce_csv(usage_statistics)
+  end
+
+private
+
+  def usage_statistics
+    questions = I18n.t("coronavirus_form.groups").map { |_, v| v[:questions] }.reduce(:merge)
+
+    start_date = Date.parse(sanitize(params[:start_date])).beginning_of_day
+    end_date = Date.parse(sanitize(params[:end_date])).end_of_day
+
+    results = {}
+    questions.each_key do |question|
+      question_text = questions.dig(question, :title)
+      counts = FormResponse.
```

Comment:
This is a real nitpick, and I'm surprised the linter didn't catch it, but periods should be at the beginning of of the next line, e.g. `.where......`